CONTRIBUTING.md

```diff
@@ -164,5 +164,13 @@ See [The Problem Submodules](#the-problem-submodules) section for what needs to
 The `build.gradle` file can just be copied from any other exercise submodule.
 See the [POLICIES doc](https://github.com/exercism/java/blob/master/POLICIES.md#starter-implementations) for an explanation of when you need to add a starter implementation.
 
+4. Check if there is canonical data for the exercise you're adding.
+This can be found at `https://github.com/exercism/problem-specifications/tree/master/exercises/EXERCISE-SLUG/canonical-data.json`.
+If there is canonical data for your exercise then you should follow this when making the tests.
+We aim to follow the canonical data as closely as possible in our tests to ensure thorough test coverage.
+If there is canonical data available you also need to create a file at `exercises/exercise-slug/.meta/version` specifying the canonical data version (e.g. `1.0.0`).
```

Comment:
Maybe just good to clarify at the end of this sentence, "canonical data version" -> "canonical data version you have implemented".